Mathlib/Geometry/Manifold/MFDeriv/Defs.lean

```diff
@@ -253,7 +253,6 @@ theorem mdifferentiableAt_iff (f : M → M') (x : M) :
   rw [MDifferentiableAt, liftPropAt_iff]
   congrm _ ∧ ?_
   simp [DifferentiableWithinAtProp, Set.univ_inter]
-  -- Porting note: `rfl` wasn't needed
```

Comment:
Does this not still indicate a problem?

Reply:
Not really. There are many subtle changes in what works by reducible rfl.